source/getting-started/advanced.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: page
+title: "Advanced Installation"
+description: "Brief advanced installation tutorials."
+date: 2015-9-1 22:57
+sidebar: false
+comments: false
+sharing: true
+footer: true
+---
+
+Here are some general tutorials on how to setup some of the more advanced deployments that are frequently requested.
+
+<div class='advanced-installs-container'>
+<input name='advanced-installs' type='radio' id='debian-install' checked>
+<input name='advanced-installs' type='radio' id='fedora-install'>
+<label class='menu-selector debian' for='debian-install'>Debian (Ubuntu, Raspbian) Daemon</label>
+<label class='menu-selector fedora' for='fedora-install'>Fedora (RHEL, CentOS) Daemon</label>
+
+<div class='advanced-installs debian'>
+**Debian Deamon**
+<p>Debian based systems, including Ubuntu and Raspbian for the Raspberry Pi use an application called Init to manage daemon services. Init will launch init scripts that are located in the directory <code>/etc/init.d/</code>. A sample init script for Debian based systems is <a href="https://raw.githubusercontent.com/balloob/home-assistant/dev/scripts/hass-daemon">maintained in this project</a>.</p>
+
+<p>To install this script, download it, tweak it to you liking, and install it by following the directions in the header. This script will setup Home Assistant to run when the system boots. To start/stop Home Assistant manually, issue the following commands:
+```bash
+sudo service hass-daemon start
+sudo service hass-daemon stop
+```
+</p>
+
+<p>When running Home Assistant with this script, the configuration directory will be located at <code>/var/opt/homeassistant</code>. This directory will contain a verbose log rather than simply an error log.</p>
+
+<p>When running daemons, it is good practice to have the daemon run under its own user name rather than the default user's name. Instructions for setting this up are outside the scope of this document.</p>
+</div> <!-- DEBIAN -->
+
+<div class='advanced-installs fedora'>
+**Fedora Daemon**
```

Comment:
Yes, it potentially could apply to all flavors of Linux. Debian based systems ship with Init now rather than systemd. It can be installed, but it is not the preferred method of managing services in Debian based systems.

The link at the top includes a few Fedora derivatives in the name to grab the reader's attention.

My thoughts on these advanced installation techniques is that they are just that. Advanced. There is no one size fits all solution unless we have one installer for every system that enforces setup procedures. I don't think we should pretend this is a copy and paste job because it simply isn't.